.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-pnpm exec lint-staged
+# pnpm exec lint-staged
```

Comment:
Should this comment be removed before the PR is merged?

Reply:
@wykhuh Yes, good catch.  Thanks.